palette/src/rgb/rgb.rs

```diff
@@ -953,14 +954,50 @@ where
     }
 }
 
+impl FromStr for Rgb {
+    type Err = Box<dyn std::error::Error>;
+
+    // parses a color hex code of format '#ff00bb' or '#abc' into a
```

Comment:
It mentions "#" being part of the format, but never checks that it's there. Better to be strict and relax the rules later, rather than the opposite. Besides, it would also be technically possible to support strings without the "#". Maybe that should be added too?

Also, capital "P" in "parses". :slightly_smiling_face: